pkg/hooks/read.go

```diff
@@ -49,7 +49,7 @@ func read(content []byte) (hook *current.Hook, err error) {
 	}
 	reader, ok := Readers[ver.Version]
 	if !ok {
-		return nil, fmt.Errorf("unrecognized hook version: %q", ver.Version)
+		reader = current.Read
```

Comment:
Thanks, I'm comforted that this is more futureproof. At the risk of annoying you even further, WDYT of:
```
    logrus.Warnf("unrecognized hook version %q; defaulting to %q reader", ver.Version, current.Version)
```
(I know it will not show by default, but could help in unforeseen errors if someone remembers to use `--log-level`)

Reply:
I'm a little iffy here. We want to support `""` and `"0.0.1"` - I'm less sure we want to support `"-1"`, `"this is not a number"`, and `"1/0"` as valid hook versions.